### ISSUE TITLE
Ubuntu Server 18.04 guac-install.sh fails due to tomcat8-common

### ISSUE THREAD

**Issue by warsix4** [MysticRyuujin/guac-install#100]
```
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 tomcat8 : Depends: tomcat8-common (>= 8.5.30-1ubuntu1) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

This is on a fresh install of Ubuntu Server 18.04. I've tried `apt purge tomcat*`, `apt clean` etc., Same issue persists.

**Comment by warsix4** [comment]
I found that adding `universe` to each line of my `/etc/apt/sources.list` and running `sudo apt update` allowed me to continue. Seems the install script did not add universe to each line and only the first.

**Comment by MysticRyuujin** [comment]
This should be fixed in #105 - I test with tomcat9 and tomcat9 is now preferred
